semaphore/.semaphore/bin/run-brakeman

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+bundle exec brakeman \
+  --run-all-checks \
+  -o /dev/stdout --color \
+  -o $BRAKEMAN_REPORT_HTML
```

Comment:
`--ignore-config config/brakeman.ignore` is automatically applied if the `config/brakeman.ignore` file is present.